privilege/privileges/privileges.go

```diff
@@ -205,6 +207,33 @@ func (p *UserPrivileges) RequestVerification(db, table, column string, priv mysq
 	return mysqlPriv.RequestVerification(user, host, db, table, column, priv)
 }
 
+// ConnectionVerification implements the Checker interface.
+func (p *UserPrivileges) ConnectionVerification(user, host string, auth, salt []byte) bool {
+	mysqlPriv := p.Handle.Get()
+	record := mysqlPriv.ConnectionVerification(user, host)
+	if record == nil {
+		log.Errorf("Get user privilege record fail: user %v, host %v", user, host)
+		return false
+	}
+
+	pwd := record.Password
+	if len(pwd) != 0 && len(pwd) != 40 {
```

Comment:
it's not a meaningful constant and it's not used anywhere else, so I think it's unnecessary to define this constant.